### PR TITLE
Export `sys::kNumUGCResultsPerPage`

### PR DESCRIPTION
**Pull request by WilliamVenner**
Useful for chunking paginated results.

### CODE REVIEW

File: src/ugc.rs [hunk 1]
```diff
@@ -9,6 +9,8 @@ use std::mem;
 use std::path::Path;
 use std::os::raw::c_char;
 
+pub use sys::kNumUGCResultsPerPage as kNumUGCResultsPerPage;
```
Comment (Thinkofname) [inline src/ugc.rs:12]: Should follow [rust style guidelines](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/advice.md#names). Also should use rust types for the public facing API.

e.g.
```suggestion
pub const RESULTS_PER_A_PAGE: u32 = sys::kNumUGCResultsPerPage as u32;
```

Comment (WilliamVenner) [reply src/ugc.rs:12]: What does the `_A_` signify? I think `RESULTS_PER_PAGE` is probably sufficient.

Comment (Thinkofname) [reply src/ugc.rs:12]: Just thought it read a bit better but I don't mind about the name too much.

**Comment by Thinkofname** [comment]
Thanks!

**Comment by WilliamVenner** [comment]
@Thinkofname Just noticed - `steamworks::RESULTS_PER_PAGE` is very ambiguous. `steamworks::ugc` is a private module, but is re-xported in main.rs so `steamworks::ugc::RESULTS_PER_PAGE` can be accessed via `steamworks::RESULTS_PER_PAGE`

I propose we change it to `steamworks::UGC_RESULTS_PER_PAGE`?

**Pull request merged**